apps/prairielearn/src/question-servers/freeform.ts

```diff
@@ -1125,7 +1126,7 @@ export async function render(
       }
 
       const extensions = context.course_element_extensions;
-      const dependencies = {
+      const dependencies: Record<string, string[]> = {
```

Comment:
This loses information about the specific keys that are present in the object. In theory, those types would be useful downstream of this. IIRC you had `as string[]` inline here before, I think that was preferable.

Reply:
Fixed in 6d3ab0659925d5cb29dd02ca81164ee829466c43.